src/2weso_test.cpp

```diff
@@ -58,10 +59,11 @@ int main() {
     proof = ProveTwoWeso(D, f, iteration, 0, &weso, 0, stopped);
     CheckProof(D, proof, iteration);
     // Test 3 - 100 million iters.
-    iteration = 100000000;
+    iteration = 30000000;
     proof = ProveTwoWeso(D, f, iteration, 0, &weso, 0, stopped);
     CheckProof(D, proof, iteration);
     // Test stopping gracefully.
     stopped = true;
     vdf_worker.join();
+    exit(0);
```

Comment:
the fact that this helps suggests we're corrupting memory somewhere. `-fsanitize=address` might help find it